UPGRADE_GUIDE.md

```diff
@@ -5,6 +5,143 @@ To see discussion around these API changes, please refer to the
 [changelog](/CHANGELOG.md) and visit the commits and issues they
 reference.
 
+0.13.x -> 1.0.0
+----------------
+
+- React `0.14.x` is now required (if you want to use the new AsyncProps feature of React Router)
+- `Router.run` has been removed. Simply call `React.render()` directly on the `<Router/>` component
+- `<RouteHandler/>` has been replaced with `this.props.children`
+
+```js
+// 0.13.x
+var SomeComponent = React.createClass({
+  render() {
+    return (
+      <div>
+        <RouteHandler customProp={true} />
+      </div>
+    )
+  }
+});
+
+// 1.0.0
+var SomeComponent = React.createClass({
+  render() {
+    return (
+      <div>
+        {this.props.children && React.cloneElement(this.props.children, {
+          customProp: true
+        })}
+      </div>
+    )
+  }
+});
+```
+
+- `this.context.router` has been replaced with a combination of `this.props.*`
+values and `this.*` methods
+
+```js
+// 0.13.x
+var SomeComponent = React.createClass({
+  contextTypes = {
+    router: React.PropTypes.func.isRequired
+  },
+
+  render() {
+    return (
+      <div>
+        Current route is: {this.context.router.getCurrentPath()}.
+        <button onClick={this._handleClick}>Click to Navigate Away</button>
+      </div>
+    )
+  },
+
+  _handleClick () {
+      this.context.router.transitionTo('login');
+  }
+});
+
+// 1.0.0
+var SomeComponent = React.createClass({
+  render() {
+    return (
+      <div>
+        Current route is: {this.props.location.pathname}
+        <button onClick={this._handleClick}>Click to Navigate Away</button>
+      </div>
+    )
+  },
+
+  _handleClick () {
+      this.transitionTo('login');
```

Comment:
nvm, I see that `RouterContextMixin` is on the Router component's `createClass`. Now to figure out, why its not wfm.